app/controllers/carts_controller.rb

```diff
@@ -6,7 +6,7 @@ def create
     @cart.add_item(item.id)
     session[:cart] = @cart.contents
     flash[:notice] = "#{item.title} added to cart"
-    redirect_to root_path
+    redirect_to items_path
```

Comment:
@meyerhoferc the root path has the jumbotron, the items path is the items display page without the jumbotron. Does that make sense? Since it's more of a welcome page I figured the user should only see it upon first getting to the site.